### PR TITLE
fix depreciation dates

### PR DESCRIPTION
**Pull request by jessfraz**
Follow up to the convo in https://github.com/docker/docker/pull/16894, before we forget

### CODE REVIEW

**Comment by jfrazelle** [comment]
ping @diogomonica :imp:

File: docs/misc/deprecated.md [hunk 1]
```diff
@@ -106,8 +106,11 @@ the path does not exist.
...
 
 ### Docker Content Trust ENV passphrase variables name change
+**Deprecated In Release: v1.9**
+
+**Target For Removal In Release: v1.10**
 
 As of 1.9, Docker Content Trust Offline key will be renamed to Root key and the Tagging key will be renamed to Repository key. Due to this renaming, we're also changing the corresponding environment variables
 
 - DOCKER_CONTENT_TRUST_OFFLINE_PASSPHRASE will now be named DOCKER_CONTENT_TRUST_ROOT_PASSPHRASE
-- DOCKER_CONTENT_TRUST_TAGGING_PASSPHRASE will now be named DOCKER_CONTENT_TRUST_REPOSITORY_PASSPHRASE
```
Comment (diogomonica) [inline docs/misc/deprecated.md]: What is this change?

**Comment by diogomonica** [comment]
LGTM

File: docs/misc/deprecated.md (hunk 1 above)
Comment (jfrazelle) [inline docs/misc/deprecated.md]: Whitespace vim plugin or some shiz

**Comment by icecrime** [comment]
LGTM

**Comment by thaJeztah** [comment]
Thanks for the fix, @jfrazelle 

LGTM

**Pull request merged**